api/v1alpha1/account_types.go

```diff
@@ -270,7 +272,7 @@ func (a *Account) GetOptInRequestsByStatus(stati OptInRequestStatus) (int, OptIn
 func (a *Account) HasOpenQuotaIncreaseRequests() bool {
 	for _, accountServiceQuotas := range a.Status.RegionalServiceQuotas {
 		for _, v := range accountServiceQuotas {
-			if v.Status != ServiceRequestCompleted {
+			if v.Status != ServiceRequestCompleted || v.Status == ServiceRequestUnknown {
```

Comment:
We can leave this line alone, as the second clause is already caught by the first one. That is, if `v.Status` is `ServiceRequestUnknown`, that will already be caught by `v.Status != ServiceRequestCompleted`

Reply:
yeah that should actually be `v.Status != ServiceRequestUnknown`   bc we want return false whenever all service request are Completed or Unknown

Reply:
Fixes